<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextEdit" name="server_te_recv">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>461</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="server_te_send">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="server_pb_send">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>340</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="server_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>输入消息</string>
   </property>
  </widget>
  <widget class="QLabel" name="server_label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>消息</string>
   </property>
  </widget>
  <widget class="QLabel" name="server_label_info">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
